docker/sweepers_driver.py

```diff
@@ -58,11 +58,22 @@
 import json
 import logging
 import os
-from typing import Callable
+from typing import Callable, Iterable
 
 from pds.registrysweepers import provenance, ancestry
+from pds.registrysweepers.utils import configure_logging
+
+configure_logging(filepath=None, log_level=logging.INFO)
+log = logging.getLogger(__name__)
+
+dev_mode = str(os.environ.get("DEV_MODE")).lower() not in {'none', '', '0', 'false'}
+if dev_mode:
+    log.warning(f'Operating in development mode - host verification disabled')
+    import urllib3
+    urllib3.disable_warnings()
 
 opensearch_endpoint = os.environ.get("PROV_ENDPOINT")
+log.info(f'Targeting base OpenSearch endpoint "{opensearch_endpoint}"')
```

Comment:
Is there a reasonable default for `PROV_ENDPOINT` we can add as a second argument to `os.environ.get` in the case someone forgets to set the env var?

Reply:
Two possible approaches:
 - throw if not set
 - default to default local-dev values for endpoint and creds
 
 Thoughts on preference?

Reply:
I'd throw. Fail fast!